.changeset/grumpy-pandas-dress.md

```diff
@@ -0,0 +1,6 @@
+---
+"@heathmont/moon-core-tw": patch
+"docs": patch
+---
+
+Add isSwipeDragDisabled property to Carousel component
```

Comment:
Please, use a PR format message with a ticket number

Reply:
Updated. Thanks for the hint.